Snowflake.Data.Tests/IntegrationTests/SFDbCommandIT.cs

```diff
@@ -1674,5 +1676,85 @@ public async Task TestCommandWithCommentEmbeddedAsync()
                 Assert.AreEqual("--", reader.GetString(0));
             }
         }
+
+        [Test]
+        public void TestExecuteNonQueryReturnsCorrectRowCountForUploadWithMultipleFiles()
+        {
+            const int NumberOfFiles = 5;
+            const int NumberOfRows = 3;
+            const int ExpectedRowCount = NumberOfFiles * NumberOfRows;
+
+            using (SnowflakeDbConnection conn = new SnowflakeDbConnection())
+            {
+                conn.ConnectionString = ConnectionString + "poolingEnabled=false";
+                conn.Open();
+
+                using (SnowflakeDbCommand cmd = (SnowflakeDbCommand)conn.CreateCommand())
+                {
+                    // Arrange
+                    var data = string.Concat(Enumerable.Repeat(string.Join(",", "TestData") + "\n", NumberOfRows));
+                    var prefix = $"{Path.GetTempPath()}{Guid.NewGuid()}";
```

Comment:
maybe we could add to the file name a part which says something about for which test it was created?

Reply:
Added the test name to the file name